plugins/comments.md

```diff
@@ -14,6 +14,27 @@ Comments is a premium plugin from Tiny. Please see the [Premium features]({{site
 
 Once you have obtained the Comments 2.0 plugin, refer to the following instructions for using it.
 
+## Add Comments plugin
+
+Following is an example to add Comments 2.0 plugin to the TinyMCE editor:
+
+```js
+tinymce.init({
+  selector: '#tiny-ui .editor',
+  toolbar: 'bold italic underline | addcomment',
```

Comment:
I think TinyComments is in the default now (check in a fiddle), so you might be able to remove this.